src/common/routing/hooks/useBackListener.js

```diff
@@ -0,0 +1,20 @@
+import { useContext, useEffect } from "react";
```

Comment:
If the hook is inspired from a source(SoF), could you please post a comment about it with a link?

Reply:
Will add it